packages/cli/src/commands/link/link.js

```diff
@@ -32,31 +32,11 @@ type FlagsType = {
  *             only that package is processed.
  */
 function link([rawPackageName]: Array<string>, ctx: ContextT, opts: FlagsType) {
-  let platforms;
-  let project;
-  try {
-    platforms = getPlatforms(ctx.root);
-    logger.debug(
-      'Available platforms: ' +
-        `${Object.getOwnPropertyNames(platforms)
-          .map(platform => getPlatformName(platform))
-          .join(', ')}`,
-    );
-    if (opts.platforms) {
-      platforms = pick(platforms, opts.platforms);
-    }
-    logger.debug(
-      'Targeted platforms: ' +
```

Comment:
can you bring back the debug logs?

Reply:
Brought them back in a different form, you should be satisfied.